### ISSUE TITLE
Old frame slabs are invisible in 0.4.0

### ISSUE THREAD

**Issue by Seika85** [PianoManu/BlockCarpentry#53]
Frame slabs placed witch 0.3.6 are still invisible in 0.4.0 as mentioned at #49 (with 0.3.11).

![2021-08-29_17 12 33_e](https://user-images.githubusercontent.com/6469794/131255530-422ecc0a-0e93-45ee-a11d-e7034df2cb74.jpg)

Newly placed frame slabs work correctly.

**Comment by PianoManu** [comment]
Thanks for reporting! I already expected something like that. The problem is, I can't change much for existing worlds. I changed the tile entities for slabs so that they can contain two blocks (for double slabs) so already existing slabs have the outdated version of the tile entity which cannot be read anymore. I might find a workaround but I guess the easiest way would be to remove the slabs in question and place them again. Sorry for the circumstances!
Regards, Manu

**Comment by Seika85** [comment]
Yeah. Well. Removing and placing them again manually is not an option as I run a local server where a lot of decorative use of those old-version slabs across many many chunks has already happened for months and months. That would be a massive task to redo all of that.
So as of now we stick with 0.3.6.

If you find a workaround or come up with a mirgration script, that would be awesome. But don't feel pressured.
Maybe add a warning message to the mod page in the meantime, so that others don't run into the same issues and hassle?

Thank you for your awesome mod. Keep up the good work.  :)

**Comment by PianoManu** [comment]
Alright, I see. But as I said, I'm very sorry, but I can't do anything about it, as far as I know. I'm not very experienced in modding, though. If anyone else could give me tips how to solve this issue, I would appreciate it a lot!
I'll add a "help wanted" tag, maybe somebody else could help...

Thanks for your kind words :)
Regards, Manu
